gufe/ligandnetwork.py

```diff
@@ -313,9 +348,11 @@ def to_rbfe_alchemical_network(
     #     )
 
     def is_connected(self) -> bool:
-        """Are all ligands in the network (indirectly) connected to each other
+        """Indicates whether all ligands in the network are (indirectly)
+        connected to each other.
+
+        A ``False`` value indicates that either some ligands have no edges or
```

Comment:
```suggestion
        A ``False`` value indicates that either some nodes have no edges or
```
Just to keep nodes/edges language together.